examples/test1/build.gradle

```diff
@@ -1,67 +1,49 @@
 /*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Based on jruyi/thrift-gradle-plugin/../examples/test1/build.gradle
+ * Modified 2023 LINE Corporation
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
  */
 
-buildscript {
-	repositories {
-		mavenLocal()
-		jcenter()
-	}
-	dependencies {
-		classpath 'org.jruyi.gradle:thrift-gradle-plugin:+'
-	}
-}
-
-apply plugin: 'java'
-apply plugin: 'org.jruyi.thrift'
-
-group = 'org.jruyi.gradle'
-archivesBaseName = 'thrift-gradle-plugin-test'
-description = 'Test thrift-gradle-plugin'
-
-ext {
-	title = 'Thrift Gradle Plugin Test'
-	organizationId = 'org.jruyi'
-	organizationName = 'JRuyi.org'
-	organizationUrl = 'http://www.jruyi.org/'
-
-	licenseName = 'Apache License, Version 2.0'
-	licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-
-	createdBy = "${System.getProperty('java.version')} (${System.getProperty('java.vm.vendor')})"
+plugins {
+    id 'java'
+    id 'com.google.osdetector' version '1.7.3'
+    id 'com.linecorp.thrift-gradle-plugin' version '+'
```

Comment:
Q: `+` means? 😅

Reply:
It includes any version.

The `examples` from original repository testing the plugin installed locally directly. 
So we can try to use `./gradlew publishToMavenLocal` and then run `./gradlew -p examples/test1 --refresh-dependencies --rerun-tasks clean check` with 8 or 17 to check it works or not. I just keep it and do manual test~

Reply:
Thanks for the explanation.